### PR TITLE
remove all guide and add support section

### PR DESCRIPTION
**Pull request by felipeZ**
## Changes
solve #56 with the following changes:
* Remove all outdated guide folder
* Create *support* section

The *guide* section mention above wasn't used to generate the new web page.

This PR partially fix the warnings reported at [Votca/xtp#609](https://github.com/votca/xtp/issues/609)

### CODE REVIEW

File: share/doc/help.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+-------
+If you find bugs, weird behaviour or just incomprehensible instructions,
+please open an issue
+`here <https://github.com/votca/xtp-tutorials/issues>`__
+
+We will try to fix it as fast as possible. For other questions, we provide an email list:
+
+`google
+group <https://groups.google.com/forum/?hl=de#!forum/votca-xtp>`__
+
```
Comment (junghans) [inline share/doc/help.rst:12]: This forum is dead. We are using `votca` for this , too.

Plus I added it to the main page as well

File: share/doc/help.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+and a slack channel:
+
+`Slack <https://votca.slack.com/messages/C7XVBE9EG/?>`__
```
Comment (junghans) [inline share/doc/help.rst:15]: We should add that to the main page as well.

**Review by junghans — CHANGES_REQUESTED** [review]

File: share/doc/help.rst (hunk 1 above)
Comment (felipeZ) [reply share/doc/help.rst:12]: Thanks for let me know!

**Review by JensWehner — APPROVED** [review]
nice work

**Review by junghans — APPROVED** [review]

**Pull request merged**